src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-prometheus.mdx

```diff
@@ -15,29 +15,25 @@ You can collect metrics from any [software that exposes Prometheus metrics](http
 
 Complete the steps below to collect Prometheus metrics.
 
-## Step 1: Sign up for your free account if you haven't already [#signup]
+## Prerequisites [#prereq]
 
-<InlineSignup />
+Before you can start, you'll need to [sign up for a New Relic account](https://newrelic.com/signup). Once you've done that, you need to make the Prometheus metrics endpoint accessible from the OpenTelemetry Collector. The collector will use the Prometheus receiver component.
 
-## Step 2: Prerequisites [#prerequisites]
-
-The collector will use the Prometheus receiver component.
-
-Depending on the software you want to monitor the prerequites may change. Prometheus metrics endpoint could be enabled by defaut or may need some setup in the configuration to enable it.
+* Depending on the software you want to monitor the prerequites may change.
+* Prometheus metrics endpoint could be enabled by defaut or may need some setup in the configuration to enable it.
 Others software can require a [third-party Prometheus exporter](https://prometheus.io/docs/instrumenting/exporters/#third-party-exporters) to expose metrics.
 
-In both cases, the Prometheus metrics endpoint needs to be accesible from the OpenTelemetry Collector.
-
-## Step 3: Install Opentelemetry collector [#install-opentelemetry-collector]
-
-If not done yet, download and install the Opentelemetry collector following [Opentelemetry docs](https://opentelemetry.io/docs/collector/getting-started/).
+<Steps>
+    <Step>
+    ## Install Opentelemetry collector
 
-Install [OpenTelemetry Collector Contrib Distro](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib) or other distribution including, at least, the following components:
+If not done yet, download and install the Opentelemetry collector following [Opentelemetry docs](https://opentelemetry.io/docs/collector/getting-started/). 
```

Comment:
@mangulonr This is the step that @thezackm called out as needing some updates. This step on its own isn't very helpful, either:

* This step is about installing the OTel Collector, but then links to Prometheus receiver and OTLP exporter. Can you explain how these two bullet points relate to the install process in OTel collector? Are they folded into the install process? Why are they separate bullet points? What are users supposed to do with these links -- just having them linked out doesn't tell us much
* I removed the Collector Contrib distro. Should I add anything about this repo that could be helpful for users? Zack mentioned that this is included out of the box as part of the vanilla install, but if someone needed to do a special configuration, would this repo be relevant?

Reply:
Something else that confuses me (and might confuse our users) is how installing the OTel collector relates to the Prometheus collector. If I'm understanding correctly:

- The Prometheus collector integrates with our existing OTel Collector
- Users get the Prometheus collector by configuring the OTel Collector
- So step #1 of getting the Prometheus collector is really step 4, "Configure OTel Collector."

The prereqs would then read:

- Sign up for an account
- Install the OTel collector
- Ensure that the OTel collector has "something" enabled so that it can access the Prometheus metrics endpoints.

Does this sound correct?

Reply:
@akristen 
Thanks for taking the initiative here.

We pretended to state that our customers could use any  OTel distribution that includes the components mentioned (Prometheus receiver and OTLP exporter). Customer can also create their own OTel distribution

In both links, you can check in which OTel distributions are those components included.

We used to recommend installing the contrib  distro because it not only contains components tied to open source projects ([like Core distribution](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol)) but also some vendor supported ones ([link](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/README.md)).

For example, in the case of other collector examples we have in our docs website like Docker and Redis we can't recommend the Core distro because it requires a vendor-specific receiver.

Maybe we can keep the recommendation of installing distro at least with the other examples but including the link to the install instructions you're proposing.

Prometheus receiver is just a component that can be included in a collector distribution it is not a collector for itself.

Also I'm checking that the sign-up widget is not displaying any content.